browser-emulator-builder/lib/domMatch.ts

```diff
@@ -35,7 +35,7 @@ export function isAllowedValueDifference(diff: IDiff): boolean {
   return false;
 }
 
-const explicitExportsToIgnore = ['window.navigator', 'window.chrome'];
+const explicitExportsToIgnore = ['window.chrome'];
```

Comment:
Navigator should now be correctly polyfilled (to be checked). This means navigator plugin can probably be cleaned up a lot. Something that should probably be patched now there is chrome features.